src/telemetry/metrics.ts

```diff
@@ -149,10 +153,27 @@ export const startMetrics = (serviceName: keyof typeof counterMap): void => {
   const meterProvider = new metrics.MeterProvider({ resource, readers });
   api.metrics.setGlobalMeterProvider(meterProvider);
 
+  const performanceMeter = meterProvider.getMeter(`api-${serviceName}-perf`);
```

Comment:
Not sure if needs to be named something else?

Reply:
Just wanted to differentiate from other metrics, but maybe `serviceName` is also fine?

Reply:
You can still use the same meterProvider, it doesn't really make that much of a difference